README.md

```diff
@@ -200,6 +200,9 @@ Usage: bats [-cr] [-f <regex>] [-j <jobs>] [-p | -t] <test>...
   -f, --filter     Filter test cases by names matching the regular expression
   -h, --help       Display this help message
   -j, --jobs       Number of parallel jobs to run (requires GNU parallel)
+  --parallel-preserve-environment 
```

Comment:
Perhaps this should be `--jobs-preseve-env` ? It relates to that flag rather than the implementation IMHO

Reply:
I see where you are coming from and have no hard preference here. A hypothetical other implementation that would not "suffer" from parallel's lack of enviroment propagation would not need this flag though.